internal/services/tesla_task_service.go

```diff
@@ -51,6 +51,7 @@ type TeslaCredentialsV2 struct {
 	AccessToken   string    `json:"accessToken"`
 	Expiry        time.Time `json:"expiry"`
 	RefreshToken  string    `json:"refreshToken"`
+	APIVersion    int       `json:"apiVersion"`
```

Comment:
It's confusing to me that it's `version` in one place, `apiVersion` in another, `teslaApiVersion` in a third. I'm going to cope and say that this is temporary and that doesn't matter much. Let's see.

Reply:
@KevinJoiner Any opinion?

Reply:
I agree. I had the same confusion. I wasn't sure if `RegisterDeviceIntegrationRequest` was only used for Tesla or not. If we also use this request for synthetic devices, then simply using the version works.